From b410254fa3d447fe45f6ee542e18e233b2f5b738 Mon Sep 17 00:00:00 2001
From: Behan Webster <behanw@converseincode.com>
Date: Fri, 14 Feb 2014 15:19:17 -0800
Subject: [PATCH] all: LLVMLinux: Change DWARF flag to support gcc and clang

Both gcc and clang accept the this version of the command line option.

Signed-off-by: Behan Webster <behanw@converseincode.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 43a0fe4..4952606 100644
--- a/Makefile
+++ b/Makefile
@@ -670,7 +670,7 @@ endif
 
 ifdef CONFIG_DEBUG_INFO
 KBUILD_CFLAGS	+= -g
-KBUILD_AFLAGS	+= -Wa,--gdwarf-2
+KBUILD_AFLAGS	+= -Wa,-gdwarf-2
 endif
 
 ifdef CONFIG_DEBUG_INFO_REDUCED
-- 
1.8.3.2

